azkaban-common/src/main/java/azkaban/imagemgmt/rampup/ImageRampupManagerImpl.java

```diff
@@ -296,11 +296,16 @@ public VersionInfo getVersionInfo(final String imageType, final String imageVers
       if (null == flow) {
         log.info("Flow object is null, so continue");
         final ImageRampup firstImageRampup = imageRampupList.get(0);
+
+        // Find the imageVersion in the Rampup list with maximum rampup percentage.
+        final ImageRampup maxImageRampup = imageRampupList.stream()
+            .max(Comparator.comparing(ImageRampup::getRampupPercentage))
+            .orElseGet(() -> firstImageRampup);
         imageTypeRampupVersionMap.put(imageTypeName,
-            this.fetchImageVersion(imageTypeName, firstImageRampup.getImageVersion())
+            this.fetchImageVersion(imageTypeName, maxImageRampup.getImageVersion())
```

Comment:
I thought we decided on using the latest active version and not the one which is max ramped up.

Reply:
In the case when there is no ramp-up strategy, then the latest active version will be displayed as that will be the one to be utilized. However, when the ramp plan is specified, then the one with the max percentage of it will be displayed. This will be consistent when a version is 100% in the ramp up plan.